CharlieBackend.AdminPanel/Services/CourseService.cs

```diff
@@ -25,12 +20,24 @@ public CourseService(IApiUtil apiUtil, IMapper mapper)
             _mapper = mapper;
         }
 
+        public async Task<CourseDto> DisableCourseAsync(long id)
+        {
+            return await
+                _apiUtil.DeleteAsync<CourseDto>($"api/courses/{id}");
+        }
+
+        public async Task UpdateCourse(long id, UpdateCourseDto UpdateDto)
+        {
+            await
+                _apiUtil.PutAsync<UpdateCourseDto>($"api/courses/{id}", UpdateDto);
+        }
+
         public async Task<IList<CourseViewModel>> GetAllCoursesAsync()
         {
-            var courses =  _mapper.Map<IList<CourseViewModel>>(await _apiUtil.GetAsync<IList<CourseDto>>($"api/courses/isActive"));
+            var courses = _mapper.Map<IList<CourseViewModel>>(await _apiUtil.GetAsync<IList<CourseDto>>($"api/courses/isActive"));
 
             return courses;
         }
 
     }
-}
+}
```

Comment:
Add line at end of the file

Reply:
Solved https://github.com/ita-social-projects/WhatBackend/pull/415/commits/1ea845a7a7f561d63b99a636f5c628ab04f9fb29